src/distrs/beta.jl

```diff
@@ -12,5 +11,7 @@ import .RFunctions:
     betainvlogccdf
 
 # pdf
-betapdf(α::Float64, β::Float64, x::Float64) = exp(betalogpdf(α, β, x))
-betapdf(α::Real, β::Real, x::Real) = betapdf(Float64(α), Float64(β), Float64(x))
+betapdf(α::Real, β::Real, x::Number) = x^(α - 1) * (1 - x)^(β - 1) / beta(α, β)
+
+# logpdf
+betalogpdf(α::Real, β::Real, x::Number) = (α - 1) * log(x) + (β - 1) * log(1 - x) - lbeta(α, β)
```

Comment:
Thank you for cc Simon.

BTW I did a stupid/quick check by `@elapsed [log1p(-x) for _ in 1:200000000]` versus `@elapsed [log(1-x) for _ in 1:200000000]` with `x = -1.2` set, which gives me `17.890346949`s for `log1p()` and `16.294338502`s for `log()` on my local machine. Does this way for checking make sense or I need to do check on random `x`s?